frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/attachmentUsages.jspf

```diff
@@ -34,7 +34,13 @@
         >
         <thead>
         <tr>
-            <th colspan="7" class="headlabel"><liferay-ui:message key="linked.releases.and.projects" /></th>
+            <th colspan="7" class="headlabel"><liferay-ui:message key="linked.releases.and.projects" />
+            <core_rt:if test="${projectList.size() > 1 or (projectList.size() == 1 and not empty projectList.get(0).linkedReleases)}">
+                (<a href="#" id="expandAll" onclick="$('#AttachmentUsagesInfo').treetable('expandAll');  return false;" class="text-primary"><liferay-ui:message key="expand.all" /> </a>|
```

Comment:
May be we should consider avoiding inline scripts.
[Ref](https://robertnyman.com/2008/11/20/why-inline-css-and-javascript-code-is-such-a-bad-thing/)

Reply:
Done.